clients/admin-ui/src/features/privacy-requests/configuration/MailgunEmailConfiguration.tsx

```diff
@@ -0,0 +1,223 @@
+import { Box, Button, Divider, Heading, Stack } from "@fidesui/react";
+import { Form, Formik } from "formik";
+import { useState } from "react";
+
+import { CustomTextInput } from "~/features/common/form/inputs";
+import { getErrorMessage } from "~/features/common/helpers";
+import { useAlert } from "~/features/common/hooks";
+import {
+  useCreateMessagingConfigurationMutation,
+  useCreateMessagingConfigurationSecretsMutation,
+  // useGetMessagingConfigurationDetailsQuery,
```

Comment:
The work was already done to test email connection before [this issue](https://github.com/ethyca/fides/issues/2237) was created. I am leaving the work commented out here for when the BE is complete and I can finish that issue.

Reply:
These changes are too large to leave in as commented blocks. When splitting work like this, moving the changes to their own commit is the best way to keep track. Then you move that commit to a new branch you'll use when working on the other issue.

So to clean this up:
1. Start on this branch (`2236/messaging-provide`)
2. Create a new branch `lkcsmith/2237`
3. Git push `lkcsmith/2237` just to make sure your changes are saved
4. Checkout `2236/messaging-provide` again
5. Delete the commented sections, commit, and push to this PR.
6. When this PR is merged and you're ready to work on 2237, start from `lkcsmith/2237` and resolve the conficts with the changes you wanted to keep.

Note that your commit history is harder to keep track of because you have a lot of commits with merges from `main` mixed in. To see your changes as a single commit, I used [this trick](https://stackoverflow.com/a/69827502) and created a branch that has all your changes in one commit:
https://github.com/ethyca/fides/commits/ssangervasi/fides/2236/messaging-provide
That might make it easier to split things.

Reply:
Thank you! Saving for future reference as well 🙏